exec/java-exec/src/test/java/org/apache/drill/test/OperatorFixture.java

```diff
@@ -351,8 +603,110 @@ public OperatorStats getStats() {
     }
   }
 
-  public OperatorContext operatorContext(PhysicalOperator config) {
-    return new TestOperatorContext(context, allocator(), config, stats);
+  public static class MockPhysicalOperator implements PhysicalOperator {
```

Comment:
Seems overkill. Most times, a test that needs an operator will provide one. The dummy one that was in this code (or maybe in my latest PR) handles those cases where we don't even have an operator (record batch) so any operator (definition) will do.

Reply:
Thanks for catching this. This class is left over from some previous work and is not used. I've deleted it.